rules/retrieve.smk

```diff
@@ -1015,59 +1015,145 @@ if (LAU_REGIONS_DATASET := dataset_version("lau_regions"))["source"] in [
         run:
             copy2(input["lau_regions"], output["zip"])
 
+
+if (SEAWATER_TEMPERATURE_DATASET := dataset_version("seawater_temperature"))[
+    "source"
+] in [
+    "archive",
+]:
+
     rule retrieve_seawater_temperature:
         params:
-            default_cutout=config_provider("atlite", "default_cutout"),
+            cutout_dict=config_provider("atlite", "cutouts"),
+        input:
+            data=storage(f"{SEAWATER_TEMPERATURE_DATASET['url']}"),
         output:
-            seawater_temperature="data/seawater_temperature_{year}.nc",
+            seawater_temperature=f"{SEAWATER_TEMPERATURE_DATASET['folder']}/seawater_temperature_{{cutout}}.nc",
```

Comment:
```suggestion
            seawater_temperature=f"{SEAWATER_TEMPERATURE_DATASET['folder']}/{{cutout}}.nc",
```

The seawater temperature name part is already in the folder. Also is the output of this rule supposed to be the same as for `build_seawater_temperature_copernicusmarine`? Because currently the output filenames are differing.